data/language/pl-PL.txt

```diff
@@ -4467,4 +4467,4 @@ STR_DTLS    :Na twoim terenie odbywa się wielki doroczny festiwal muzyczny. Wyb
 <Rock 'n' Roll - Rock 'n' Roll>
 STR_SCNR    :Rock 'n' Roll - Rock 'n' Roll
 STR_PARK    :Rock 'n' Roll Revival
-STR_DTLS    :Ten starzejący się lunapark ma najlepsze dni za sobą. Pomóż właścicielom nadać mu stylowy, rockandrollowy wygląd i przekształć to miejsce w dochodowe przedsięwzięcie.
\ No newline at end of file
+STR_DTLS    :Ten starzejący się lunapark ma najlepsze dni za sobą. Pomóż właścicielom nadać mu stylowy, rockandrollowy wygląd i przekształć to miejsce w dochodowe przedsięwzięcie.
```

Comment:
This EOL change will need to be reverted as well, otherwise we might still run into problems with the nightly localisation merge.

Reply:
I've checked this file in repository and this line looks exactly the same. I will re-upload this file from original repository.